src/Netsphere.Resource/Scene/SceneContainer.cs

```diff
@@ -183,7 +183,7 @@ public void Serialize(Stream stream)
         {
             using (var w = stream.ToBinaryWriter(true))
             {
-                //w.Write(Version);
+                w.Write(Version);
```

Comment:
Is this actually needed? I removed Version because it caused some issues in the past for me.